docs/src/main/paradox/user/lang/dart/eventsourced.md

```diff
@@ -0,0 +1,102 @@
+# Event sourcing
```

Comment:
The original text in the java support docs has been updated. It may be good to base this on the updated text. See master and the changes in #273.

Reply:
Thanks @pvlugter